cmd/insprctl/cli/init.go

```diff
@@ -34,11 +35,13 @@ var initCommand = cmd.NewCmd("init").
 	).NoArgs(
 	func(c context.Context) error {
 		config := insprConfiguration{}
-		fmt.Print("enter insprd host (http://localhost:8080):")
+		fmt.Print("enter insprd IP (http://localhost:8080):")
```

Comment:
maybe `enter insprd IP or URL` would fit best
the example `http://localhost:8080` could just be `localhost:8080`